src/scripts/dependencies/el6.sh

```diff
@@ -12,114 +12,114 @@
 #
```

Comment:
Folks, refactoring is not a working code destruction. Some needed working code and deps installations were removed. This should not have happened. @person124 @matthewroy01